### PR TITLE
Modification for sommelier-3

### PR DESCRIPTION
**Pull request by IbrarMakaveli**
Closes: #XXX

### CODE REVIEW

**Review by zmanian — APPROVED** [review]

**Comment by clemensgg** [comment]
changes needed to make it work with `gorc v0.3.9` 

- correct sign-delegate-keys command syntax: (option -a must be used in front of every argument)
```sh
gorc sign-delegate-keys -a $key_name -a $val_address
``` 

- output of gorc sign-delegate-keys is the correct ETH signature but missing the "0x" prefix, which needs to be added manually after

example output:
```sh
gorc --config $HOME/gorc/config.toml sign-delegate-keys -a signer -a sommvaloperxxxxx cb43830c2af4cc2397411ea08aed6e384a5eae78d456fa75965b951295bb74fa0098efa381c2fb32                                                                                                             8b8449b0c514ff255d11bc60c8fed38ef9561771b71598c91b
```

but sommelier needs:
```
0xcb43830c2af4cc2397411ea08aed6e384a5eae78d456fa75965b951295bb74fa0098efa381c2fb32                                                                                                             8b8449b0c514ff255d11bc60c8fed38ef9561771b71598c91b
```

File: README.md [hunk 1]
```diff
@@ -37,13 +37,13 @@ The Gravity Bridge requires some additional pieces to be deployed to support it:
 mkdir install && cd install
 
 # Install Orchestrator
-wget https://github.com/PeggyJV/gravity-bridge/releases/download/v0.1.21/contract-deployer https://github.com/PeggyJV/gravity-bridge/releases/download/v0.1.21/orchestrator https://github.com/PeggyJV/gravity-bridge/releases/download/v0.1.21/relayer https://github.com/PeggyJV/gravity-bridge/releases/download/v0.1.21/gorc && chmod +x * && sudo mv * /usr/bin
+wget https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.8/contract-deployer https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.8/orchestrator https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.8/relayer https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.8/gorc && chmod +x * && sudo mv * /usr/bin
```
Comment (EricBolten) [inline README.md:40]: Only the gorc binary should be necessary for the orchestrator now, so we can delete the other `wget` commands and update the version to `0.3.9`.

File: README.md [hunk 2]
```diff
@@ -37,13 +37,13 @@ The Gravity Bridge requires some additional pieces to be deployed to support it:
...
 
 # Install Geth
 wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.10.4-aa637fd3.tar.gz && tar -xvf geth-linux-amd64-1.10.4-aa637fd3.tar.gz && sudo mv geth-linux-amd64-1.10.4-aa637fd3/geth /usr/bin/geth && rm -rf geth-linux-amd64-1.10.4-aa637fd3*
 
 # Install Sommelier
-wget https://github.com/PeggyJV/sommelier/releases/download/v2.0.0/sommelier_2.0.0_linux_amd64.tar.gz && tar -xf sommelier_2.0.0_linux_amd64.tar.gz && sudo mv sommelier /usr/bin && rm -rf sommelier_2.0.0_linux_amd64* LICENSE README.md
+wget https://github.com/PeggyJV/sommelier/releases/download/v3.0.7/sommelier_3.0.7_linux_amd64.tar.gz && tar -xf sommelier_3.0.7_linux_amd64.tar.gz && sudo mv sommelier /usr/bin && rm -rf sommelier_3.0.7_linux_amd64* LICENSE README.md
```
Comment (EricBolten) [inline README.md:46]: Will have to update this to `3.1.1`.

**Review by EricBolten — COMMENTED** [review]
Hey Ibrar, thanks for this PR. I've looked it over, and it seems solid aside from some small changes and a few versions we'll have to bump for the current network. The issue that @clemensgg raised around the CLI args shouldn't be an issue on the latest version of `gorc`, but he is correct that we need to instruct people to prepend the `0x` prefix on the latest version.

File: README.md (hunk 2 above)
Comment (IbrarMakaveli) [reply README.md]: Corrected

File: README.md (hunk 1 above)
Comment (IbrarMakaveli) [reply README.md]: Corrected

**Pull request merged**